tests/system/action/meeting/test_clone.py

```diff
@@ -530,3 +531,72 @@ def test_permissions_missing_payload_committee_permission(self) -> None:
             "Missing CommitteeManagementLevel: can_manage for committee 2",
             response.json["message"],
         )
+
+    def test_clone_with_created_topic_and_agenda_type(self) -> None:
+        self.set_models(self.test_models)
+
+        response = self.request(
+            "topic.create",
+            {
+                "meeting_id": 1,
+                "title": "test",
+                "agenda_type": AgendaItem.INTERNAL_ITEM,
+                "agenda_duration": 60,
+            },
+        )
+        self.assert_status_code(response, 200)
+        topic_fqid = f'topic/{response.json["results"][0][0]["id"]}'
+        topic = self.get_model(topic_fqid)
+        self.assertNotIn("agenda_type", topic)
+        self.assertNotIn("agenda_duration", topic)
+        agenda_item_fqid = f"agenda_item/{topic.get('agenda_item_id')}"
+        self.assert_model_exists(
+            agenda_item_fqid,
+            {
+                "type": AgendaItem.INTERNAL_ITEM,
+                "duration": 60,
+                "content_object_id": topic_fqid,
+            },
+        )
+
+        response = self.request("meeting.clone", {"meeting_id": 1})
+        self.assert_status_code(response, 200)
+
+    def test_clone_with_created_motion_and_agenda_type(self) -> None:
+        self.set_models(self.test_models)
+        response = self.request(
```

Comment:
Here the request is needed, because it includes other_actions calls with data.user_id